docs/main/origins.md

```diff
@@ -9,10 +9,10 @@ last_updated: Sun Jan 13 18:35:14 2019 UTC
 Pi-hole being a **advertising-aware DNS/Web server**, makes use of the following technologies:
 
 - [`dnsmasq`](http://www.thekelleys.org.uk/dnsmasq/doc.html) - a lightweight DNS and DHCP server
-- [`curl`](https://curl.haxx.se) - A command line tool for transferring data with URL syntax
-- [`lighttpd`](https://www.lighttpd.net) - webserver designed and optimized for high performance
-- [`php`](https://secure.php.net) - a popular general-purpose web scripting language
-- [AdminLTE Dashboard](https://github.com/almasaeed2010/AdminLTE) - premium admin control panel based on Bootstrap 3.x
+- [`curl`](https://curl.haxx.se/) - A command line tool for transferring data with URL syntax
+- [`lighttpd`](https://www.lighttpd.net/) - webserver designed and optimized for high performance
+- [`php`](https://secure.php.net/) - a popular general-purpose web scripting language
+- [AdminLTE Dashboard](https://github.com/ColorlibHQ/AdminLTE) - premium admin control panel based on Bootstrap 3.x

```

Comment:
And it was with this change that I realised AdminLTE belonged to someone else now. Maybe. https://github.com/ColorlibHQ/AdminLTE/issues/2081

Reply:
Yup, so feel free to push any removals if you notice something you don't like. I noticed quite a few redirects and fixed them here.

Reply:
All good, happy to approve this one